gio/gio.go

```diff
@@ -0,0 +1,52 @@
+// +build !cgocheck
+
+package gio
+
+// #include "gio.go.h"
+// #cgo pkg-config: gtk+-2.0
+import "C"
+
+type GFile struct {
+	GFile *C.GFile
+}
+
+func cfree(s *C.char)            { C.freeCstr(s) }
+func cstring(s *C.gchar) *C.char { return C.toCstr(s) }
+func gostring(s *C.gchar) string { return C.GoString(cstring(s)) }
+
+func NewGFileForPath(filename string) *GFile {
+	ptrFilename := C.CString(filename)
+	defer cfree(ptrFilename)
+
+	return &GFile{C.g_file_new_for_path(ptrFilename)}
+}
+
+func (f *GFile) QueryInfo() *GFileInfo {
+	ptr := C.CString("standard::*")
+	defer cfree(ptr)
+
+	// C.CString(C.G_FILE_ATTRIBUTE_STANDARD_TYPE)
+	return &GFileInfo{C.g_file_query_info(f.GFile, ptr, C.G_FILE_QUERY_INFO_NONE, nil, nil)}
```

Comment:
please export `standard::*` as attributes and G_FILE_QUERY_INFO_NONE to be given from user.